sys-auth/sssd/sssd-2.0.0.ebuild

```diff
@@ -0,0 +1,208 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
```

Comment:
6

Reply:
Why?

Reply:
You're commenting on an outdated comment.